CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Versions
 2020-10-31
 -----------
 * Upgrade terraform to 0.13.4 and terragrunt to 0.25.3
+* fix aws image dependancies
```

Comment:
`dependencies` + add how you fixed it. The aim is to remind us to delete this when pip officialy changes its way to install deps.